.github/workflows/build.yml

```diff
@@ -0,0 +1,170 @@
+name: Build FreeOrion and run unit tests
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - master
+jobs:
+  build-linux:
+    name: Build FreeOrion on Ubuntu 18.04 (Bionic)
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        id: build-linux-install-cache-id
+        with:
+          path: ${{ runner.temp }}/cache-build-linux-install
+          key: build-linux-install-cache-key-v1.0
+      - uses: airvzxf/cache-anything-new-action@v1.0
+        with:
+          script: 'install-build.sh'
+          is_cached: ${{ steps.build-linux-install-cache-id.outputs.cache-hit }}
+          cache: ${{ runner.temp }}/cache-build-linux-install
+          snapshot: '/'
+          exclude: '/boot /data /dev /mnt /proc /run /sys'
+      - name: Prepare ccache timestamp
+        id: build-linux-ccache-timestamp
+        shell: cmake -P {0}
+        run: |
+          string(TIMESTAMP current_date "%Y-%m-%d-%H;%M;%S" UTC)
+          message("::set-output name=timestamp::${current_date}")
+      - name: ccache files
+        uses: actions/cache@v1.1.0
+        with:
+          path: ${{ runner.temp }}/ccache
+          key: build-linux-ccache-${{ steps.build-linux-ccache-timestamp.outputs.timestamp }}
+          restore-keys: |
+            build-linux-ccache-
+      - name: Build
+        env:
+          CACHE_NAME: linux
+          CCACHE_DIR: ${{ runner.temp }}/ccache
+          CMAKE_BUILD_PARALLEL_LEVEL: 2
+        run: |
+          echo "----------------------------------------"
+          ccache -s || true
```

Comment:
Let's use the full name for arguments in that script. This increases readability for people who is not familiar with that tool.   
`--show-stats`

Reply:
Fixed.